properties.schema

```diff
@@ -162,6 +294,58 @@
             "default": "",
             "help": "Defines the button text / icon colour when hovered over. Should be a colour that provides good contrast against the button hover colour."
           },
+          "btn-color-focus": {
+            "title": "Button colour - focus",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": ""
+          },
+          "btn-color-inverted-focus": {
+            "title": "Button colour - inverted focus",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": "",
+            "help": "Defines the button text / icon colour when focused. Should be a colour that provides good contrast against the button focus colour."
+          },
+          "btn-color-selected": {
+            "title": "Button colour - selected",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": ""
+          },
+          "btn-color-inverted-selected": {
+            "title": "Button colour - inverted selected",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": "",
+            "help": "Defines the button text / icon colour when selected. Should be a colour that provides good contrast against the button selected colour."
+          },
+          "btn-color-locked": {
+            "title": "Button colour - locked",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": ""
+          },
+          "btn-color-inverted-locked": {
+            "title": "Button colour - inverted locked",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": "",
+            "help": "Defines the button text / icon colour when locked. Should be a colour that provides good contrast against the button locked colour."
+          },
+          "disabled": {
+            "title": "Disabled colour",
```

Comment:
Update button title to _Button colour - locked_

Reply:
This should now be resolved